mod_modular_vanilla/hooks/skills/skill_container.nut

```diff
@@ -0,0 +1,47 @@
+// MV: Added
+// Part of the actor.interrupt framework
+::MSU.Skills.addEvent("onActorInterrupted", function( _offensive, _defensive ) {});
```

Comment:
I dislike how this hides the event function declaration, that you would usually expect in the `skill.nut`.
One way I mod is that I look at the base class to see what it offers. When I work on skills, I am usually only interested in what the skill.nut of vanilla or my framework mods do.

Now I also have to look in the skill_container to learn the full feature-set of skill.nut. Or where ever else that event is added.